usmqe_tests/api/grafana/test_host_dashboard.py

```diff
@@ -0,0 +1,227 @@
+
+    # get CPU Utilization panel from first row
+    panels = dashboard_rows[0]["panels"]
+    cpu_panels = [
+        panel for panel in panels
+        if "title" in panel and
+        panel["title"] == "CPU Utilization"]
+    pytest.check(len(cpu_panels) == 1)
+
+    """@pylatest grafana/cpu_utilization
+    .. test_step:: 2
+        Send **GET** request to ``GRAPHITE/render?target=[target]&format=json``
+        where [target] is part of uri obtained from previous GRAFANA call.
+        There should be target for CPU utilization of a host.
+        Compare number of hosts from Graphite with value retrieved from
+        ``workload_cpu_utilization`` fixture.
+    .. test_result:: 2
+        JSON structure containing data related to CPU utilization is similar
+        to values set by ``workload_cpu_utilization`` fixture in given time.
+    """
+    # get graphite target pointing at data containing number of host
+    target = cpu_panels[0]["targets"][0]["target"]
+    target = target.replace("$cluster_id", cluster_identifier)
+    target = target.replace("$host_name", pytest.config.getini(
+        "usm_cluster_member").replace(".", "_"))
+    target = target.strip("aliasSub(groupByNode(")
+    target = target.split("},", 1)[0]
+    target_base, target_options = target.rsplit(".{", 1)
+    target_options = target_options
+    LOGGER.debug("target: {}".format(target))
+    LOGGER.debug("target_base: {}".format(target_base))
+    LOGGER.debug("target_options: {}".format(target_options))
+    pytest.check(
+        target_options == "percent-user,percent-system",
+        "The panel CPU Utilization is composed of user and system parts")
+    target_user, target_system = ["{}.{}".format(target_base, x) for x
+                                  in target_options.split(",")]
+    # make sure that all data in graphite are saved
+    time.sleep(2)
+    graphite_user_cpu_data = graphite.get_datapoints(
+        target_user,
+        from_date=int(workload_cpu_utilization["start"].timestamp()),
+        until_date=int(workload_cpu_utilization["end"].timestamp()))
+    graphite_system_cpu_data = graphite.get_datapoints(
+        target_system,
+        from_date=int(workload_cpu_utilization["start"].timestamp()),
+        until_date=int(workload_cpu_utilization["end"].timestamp()))
+    graphite_user_cpu_data = [x for x in graphite_user_cpu_data if x[0]]
+    graphite_system_cpu_data = [x for x in graphite_system_cpu_data if x[0]]
+    graphite_user_cpu_mean = sum(
+        [x[0] for x in graphite_user_cpu_data]) / max(
+            len(graphite_user_cpu_data), 1)
+    graphite_system_cpu_mean = sum(
+        [x[0] for x in graphite_system_cpu_data]) / max(
+            len(graphite_system_cpu_data), 1)
+    expected_sample_rate =\
+        workload_cpu_utilization["end"] - workload_cpu_utilization["start"]
+    expected_sample_rate =\
+        round(expected_sample_rate.total_seconds() / 60) * SAMPLE_RATE
```

Comment:
This `expected_sample_rate` looks like expected number of datapoints rather than sample rate, could we rename it for the sake of readability?